### PR TITLE
2 comits for study purpose

### PR DESCRIPTION
**Pull request by magicalriver**
Changed the delayed time

### CODE REVIEW

File: blink.ino [hunk 1]
```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
-  digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
   delay(2000);               // wait for a second
+  digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
+  delay(5000);               // wait for a second
```
Comment (magicalriver) [inline blink.ino]: Can we do it 4 seconds

**Comment by magicalriver** [comment]
Test for general comments on pull request.

File: blink.ino [hunk 2]
```diff
@@ -20,5 +20,5 @@ void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
   delay(1000);               // wait for a second
```
Comment (magicalriver) [inline blink.ino]: I thought it should be good.

File: blink.ino (hunk 1 above)
Comment (magicalriver) [reply blink.ino]: Yes we can

**Comment by magicalriver** [comment]
I have changed my code accordingly

**Comment by nguyenhoang4875** [comment]
hello :)) 
have you nice day .

**Pull request closed without merging**